pydantic/validators.py

```diff
@@ -121,8 +121,14 @@ def float_validator(v: Any) -> float:
 
 def number_multiple_validator(v: 'Number', field: 'Field') -> 'Number':
     field_type: ConstrainedNumber = field.type_  # type: ignore
-    if field_type.multiple_of is not None and v % field_type.multiple_of != 0:  # type: ignore
-        raise errors.NumberNotMultipleError(multiple_of=field_type.multiple_of)
+    if field_type.multiple_of is not None:
```

Comment:
I think better to use `almost_equal_floats` from `utils`.

Reply:
I don't follow on this point, can you elaborate? Where would that fit?

Reply:
The use of Decimal here is unfortunate, and not great to look at, but `%` on floats in python doesn't always add up to reasonable values, e.g. `0.3 % 0.1 == 0.09999999999999998`.

I originally had an epsilon check, but it failed in those cases.

Reply:
how about something like

```py
mod = v / field_type.multiple_of % 1
if almost_equal_floats(mod, 0) or almost_equal_floats(mod, 1):
    ...
````

Reply:
I had to cast the types to float to avoid a test failure in Decimal validation, but otherwise this worked like a charm 👍